### ISSUE TITLE
Better Roll Table compatabilty

### ISSUE THREAD

**Issue by xlilcasper** [jopeek/fvtt-loot-sheet-npc-5e#32]
Increase capability with Better Roll Tables (https://github.com/ultrakorne/better-rolltables). It would be great to be able to setup it up so my vendor has a 80% for a common, 15% for an uncommon, and 5% for a rare. Roll 1d10 times. Working with better roll tables would make this easy to setup since I can have 1 table roll on another. I would make a list of common, uncommon, and rare items. Then a village vendor could have a table that makes it so he never has a rare, a city vendor could have different percent's again, and a castle could have a high chance. I hope I explained the desire well enough. 
**tl;dr; Compatibility with Better roll tables so it can roll on more than one table at a time so I can chain table rolls**

**Comment by xlilcasper** [comment]
Looks like there was an update to add an api to Better Roll Tables. This should make it very easy to integrate. See URL here: https://github.com/ultrakorne/better-rolltables/wiki/API-for-macros-and-modules#how-to-roll-tables-from-macros
